crypto/ct/ct_sct.c

```diff
@@ -223,6 +237,54 @@ int SCT_set1_signature(SCT *sct, const unsigned char *sig, size_t sig_len)
     return 1;
 }
 
+int SCT_set1_serialized_signature(SCT *sct, const unsigned char **in,
```

Comment:
(And in `i2o`, the whole pattern where the serialization method is overloaded for returning just the length, and the caller doesn't declare the available size in `out` explicitly is brittle. Sigh. Not your fault.)